addons/casings/stringtable.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="Casings">
+        <Key ID="STR_ACE_Casings_Settings_DisplayName">
+            <English>ACE Casings</English>
+        </Key>
+        <Key ID="STR_ACE_Casings_displayName">
+            <English>Casings Enabled</English>
+        </Key>
+        <Key ID="STR_ACE_Casings_description">
+            <English>Enable persistent casings</English>
```

Comment:
I feel like this needs to be elaborated a bit more. The layman will not know what this exactly means. E.g. say that it will add new objects on the ground and it might be a performance hazard for older systems.

Reply:
Added a warning, will see if anyone has better wording. All the current wording was listed from the existing PR so might need a general review.